.buildkite/pipeline.yml

```diff
@@ -32,11 +32,18 @@ steps:
               "--project-id", "catalogue_api",
               "--confirm",
               "release-deploy",
-              "--from-label", "latest",
+              "--from-label", "ref.$BUILDKITE_COMMIT",
               "--environment-id", "stage",
               "--description", $BUILDKITE_BUILD_URL,
               "--confirmation-wait-for", 3600]
   - wait
   - label: smoke tests (stage)
-    branches: "main"
+    #branches: "main"
```

Comment:
Commented out for testing.

Reply:
I generally just add the test branch name. Either way works.